src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -478,6 +478,14 @@ HttpRequestMessage FinalRequestFunc()
             {
                 // Get the request using the Func as same request cannot be resent when retries are implemented.
                 var request = requestFunc.Invoke();
+
+                // add token for dapr api token based authentication
+                var daprApiToken = Environment.GetEnvironmentVariable("DAPR_API_TOKEN");
```

Comment:
Could this value be cached?

Reply:
Fetching it from env every time allows for it to be used on platforms where an env could be refreshed without a process restart.